tests/libtest-core.sh

```diff
@@ -0,0 +1,139 @@
+# Core source library for shell script tests; this
+# file is intended to be the canonical source, which at
```

Comment:
Hmm, we should probably explicitly link to the ostree repo in the introductory blurb. Otherwise, it's impossible to tell where the canonical source actually is.